apps/chat/src/components/Chat/ModelList.tsx

```diff
@@ -111,6 +111,7 @@ const ModelGroup = ({
                 : currentEntity.name}
             </span>
             <ModelVersionSelect
+              className="absolute right-3 !h-auto"
```

Comment:
It's not a good practice to use **!important**.
Could it be avoided if possible?